runtime-display/weston/spec

```diff
@@ -1,4 +1,4 @@
-VER=10.0.2
+VER=12.0.93
```

Comment:
.93 is an unstable version (RC1 according to upstream), use 12.0.2 instead.

Reply:
Fixed.